notifier/core/params.py

```diff
@@ -0,0 +1,135 @@
+import os
+import sys
+import json
+from .cli import CLI
+from .config import Config
+
+
+class Params:
+    def __init__(self):
+        self.__default = {
+            "plugin_path": "/etc/notifier/plugins",
+            "broker_url": "",
+            "queue_name": "notifier_queue",
+            "hostname": "notifier",
+            "num_workers": 1,
+            "log_from": "INFO",
+            "log_path": "/var/logs",
+            "pid_path": "/var/run"
+        }
+        self.__cli = CLI()
+        self.__args, unk = self.__cli.parse_known_args()
```

Comment:
If we use `self.__cli.parse_args()` instead of `self.__cli.parse_known_args()` we get an error because the Celery start method arguments are not defined in our arguments.
But using `parse_known_args()` we don't get the help message when there are wrong arguments like `plugin_paht` (when should be `plugin_path`)

Reply:
This was solved in this commit: https://github.com/humu1us/notifier/pull/8/commits/b36f2fd33ee3eacb8f4ed2a3d3f819dd0f4c05f7

Reply:
Cool but this solution has a very little gotcha. When you print the help using `--help` you the following
```
$ python -m notifier --help
usage: __main__.py [-h] [-p PLUGIN_PATH] [-b BROKER_URL] [-q QUEUE_NAME]
                   [-H HOSTNAME] [-w NUM_WORKERS] [-l LOG_FROM] [-lp LOG_PATH]
                   [-pp PID_PATH] [--default-params]
                   ...

positional arguments:                  <----  Why this?
  ignored

optional arguments:
  -h, --help            show this help message and exit
  -p PLUGIN_PATH, --plugin-path PLUGIN_PATH
                        Set the plugin path
  -b BROKER_URL, --broker-url BROKER_URL
                        Set the broker url
  -q QUEUE_NAME, --queue-name QUEUE_NAME
                        Set the Celery queue name
  -H HOSTNAME, --hostname HOSTNAME
                        Set the hostname for the workers
  -w NUM_WORKERS, --num-workers NUM_WORKERS
                        Set the Celery worker number
  -l LOG_FROM, --log-from LOG_FROM
                        Set the logger level
  -lp LOG_PATH, --log-path LOG_PATH
                        Set the log file path
  -pp PID_PATH, --pid-path PID_PATH
                        Set the pid file path
  --default-params      Show default parameters
```
You can see the message ```positional arguments:
  ignored``` what is this?